### PR TITLE
ref(fetch): run helm update if chart not found 

### PR DESCRIPTION
**Pull request by michelleN**

### CODE REVIEW

**Comment by michelleN** [comment]
waiting for https://github.com/helm/helm/pull/377 to be merged before merging this.

File: action/fetch.go [hunk 1]
```diff
@@ -53,9 +53,18 @@ func fetch(chartName, lname, homedir, chartpath string) {
 	src := helm.CacheDirectory(homedir, chartpath, chartName)
 	dest := helm.WorkspaceChartDirectory(homedir, lname)
 
-	if fi, err := os.Stat(src); err != nil {
-		log.Die("Chart %s not found in %s", lname, src)
-	} else if !fi.IsDir() {
+	fi, err := os.Stat(src)
+	if err != nil {
+		log.Warn("Oops. Look like there was an issue finding the chart, %s, n %s. Running the `helm updae` to ensure you have the latest version of all Charts from Github...", lname, src)
```
Comment (mboersma) [inline action/fetch.go]: Typo: should be "Running `helm update` to ensure..."

Comment (mboersma) [inline action/fetch.go]: Also should be "Look**s** like there was..."

Comment (michelleN) [inline action/fetch.go]: Good eye! @mboersma

**Pull request merged**